pkg/scheduler/core/unit_scheduler/unit_scheduler.go

```diff
@@ -667,6 +667,7 @@ func (gs *unitScheduler) updateFailedScheduleUnit(scheduleUnit framework.Schedul
 func (gs *unitScheduler) applyToCache(ctx context.Context, unitInfo *core.SchedulingUnitInfo, result *core.UnitResult) bool {
 	cache, switchType, subCluster := gs.Cache, gs.switchType, gs.subCluster
 	successfulPods := sets.NewInt()
+	maxToleratedFailureCount := unitInfo.AllMember - unitInfo.MinMember - len(result.FailedPods)
```

Comment:
remove this variables, as well as `successfulPods`

Reply:
addressed comment, PTAL